img_height: 720
img_width: 1280

camera_matrix:
- [646.981, 0.0, 643.705]
- [0.0, 646.981, 360.907]
- [0.0, 0.0, 1.0]

distortion_coefficients: [0, 0, 0.000, 0.000000, 0.0]
